go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -149,8 +149,9 @@ func (vttablet *VttabletProcess) Setup() (err error) {
 		}
 	}()
 
+	time.Sleep(10 * time.Second)
```

Comment:
10 seconds is too much. If this is a local issue that is not happening in CI, we should not add this.

Reply:
I have instead increased `vttabletStateTimeout` to 60 seconds.